btp/provider/datasource_subaccount_destination_fragment.go

```diff
@@ -50,7 +50,14 @@ func (ds *subaccountDestinationFragmentDataSource) Schema(_ context.Context, _ d
 		MarkdownDescription: `Gets details about a specific subaccount destination fragment.
 
 __Tip:__
-You must be assigned to the admin or viewer role of the subaccount.`,
+You must be assigned admin role of the subaccount and destination service.
```

Comment:
According to https://help.sap.com/docs/connectivity/sap-btp-connectivity-cf/user-roles?q=role+collection there are more roles applicable that have less access but are fully. suffcient. Can you check?